rules-tests/Php71/Rector/FuncCall/RemoveExtraParametersRector/Fixture/skip_func_get_all_on_func_call.php.inc

```diff
@@ -1,12 +1,19 @@
 <?php
 
-namespace Rector\Php71\Tests\Rector\FuncCall\RemoveExtraParametersRector\Fixture;
+namespace Rector\Tests\Php71\Rector\FuncCall\RemoveExtraParametersRector\Fixture;
+
+if (! function_exists('\Rector\Tests\Php71\Rector\FuncCall\RemoveExtraParametersRector\Fixture\func_call_has_variadic_parameters')) {
+    function func_call_has_variadic_parameters(): array
+    {
+        return func_get_args();
+    }
+}
```

Comment:
Why is this wrapper needed for this test case?

Reply:
This place is a bit strange, if I remove the `function_exists` check, it will not be able to reproduce.

Reply:
Interesting. It might be related to PHPStan inability to read the function reflection inside this wrapper then